ansible/roles/ocp4-workload-mlops/tasks/install-jupyterhub.yaml

```diff
@@ -4,11 +4,14 @@
     state: present    
     definition: "{{ lookup('file', 'jupyterhub_kfdef.yaml' ) | from_yaml }}"
 
-- name: Add repo for jupyterhub users
-  include_tasks: add-git-repo-jupyterhub.yaml
-  loop: "{{users}}"
-  loop_control:
-    loop_var: user
-
-
+- name: "Wait for jupyterhub pod to be in Ready state"
+  k8s_info:
+    kind: Pod
+    label_selectors:
+      - 'app=jupyterhub'
```

Comment:
Please go through all of your files and maintain consistency (regardless of functionality). You have some labels that are like this and others that are like

```
label_selectors:
  - key = value
```